your-account/partial-search-staff-ministries.html

```diff
@@ -0,0 +1,13 @@
+<div class="panel-benefit panel">
+  <div class="panel-body">
+    <div class="modal-body">
+      <h1 class="banner-title">Search Staff & Ministries</h1>
+        <p class="gray">Ready to make a contribution to a new cause? Search below by staff name, staff number, Ministry name, or Quick Code:</p>
+        <div class="form-group">
+          <label for=""></label>
+          <input type="" class="form-control form-control-subtle input-lg" id="" placeholder="">
```

Comment:
@Leonardgrr this looks like out of date text:
Ready to make a contribution to a new cause? Search below by staff name, staff number, Ministry name, or Quick Code: